rest/route/project.go

```diff
@@ -414,10 +414,12 @@ func (h *projectIDPatchHandler) Run(ctx context.Context) gimlet.Responder {
 }
 
 func (h *projectIDPatchHandler) hasCommitQueuePatchDefinition(pRef *model.APIProjectRef) (bool, error) {
+	aliasesToDelete := map[string]bool{}
 	for _, alias := range pRef.Aliases {
 		if model.FromAPIString(alias.Alias) == evergreen.CommitQueueAlias {
 			return true, nil
```

Comment:
what if `alias.Delete == true`?

Reply:
These are the new aliases being added, so I'm not concerned with alias.Delete being true (technically they could do it haha but that's legitimately crazy and we don't case for that elsewhere)

Reply:
I don't understand: how else would one delete an alias?

Reply:
actually no couldn't be done (I was thinking if they added and deleted the alias in the same call then we wouldn't catch it, but that's not even possible since deleting requires an ID which requires it to be previously added)

Reply:
Let's discuss this in person